tutorials/creating-a-simple-tool.md

```diff
@@ -1 +1,7 @@
-[View tutorial here](../develop/spatial-tools/tutorial.md)
\ No newline at end of file
+---
+layout: doc
+title: Creating a Simple Tool
+permalink: /docs/tutorials/creating-a-simple-tool
+---
+
+[View tutorial here](../develop/spatial-tools/tutorial.md)
```

Comment:
Is there a way this menu item can actually just link to develop/spatial-tools/tutorial.md directly instead of me having this intermediate page? Not an issue for this PR, just thought of it for the future

Reply:
Yes, this line https://github.com/ptcrealitylab/Spatial-Toolbox-WebPage/blob/master/_data/nav.yml#L49 is the one to change